### ISSUE TITLE
Home screen/ Nav bar

### ISSUE THREAD

**Issue by Jood80** [GSG-K3/ca-application-website#49]
#

## Acceptance Criteria:
- [x]  Academy logo.
- [x] tabs 
  - [x] when signed Out ( Home, About, Steps, Mentors, Contact, Sign in)
  - [x] when signed In (Home, About, Steps, Mentors, Contacts, Notification, User Image, his/her name )
---
- [x] React Route : `/home`

- [x] Figma
- when signed out
![image](https://user-images.githubusercontent.com/56412800/80025309-1737ca00-84e9-11ea-9caf-f53085dc23e1.png)

- when signed in
![image](https://user-images.githubusercontent.com/56412800/80026018-210dfd00-84ea-11ea-8691-a9ef216d6ab3.png)

**Comment by karmelyoei** [comment]
I have this Error and I have tried to find solutions for this problem but I need your suggestion 

Error: Invalid hook call. Hooks can only be called inside of the body of a function component. This could happen for one of the following reasons:
1. You might have mismatching versions of React and the renderer (such as React DOM)
2. You might be breaking the Rules of Hooks
3. You might have more than one copy of React in the same app
See https://fb.me/react-invalid-hook-call for tips about how to debug and fix this problem.

so the first problem was with due to the React only allow to use the hooks from its library so I have to change the class component to functional component. I did and it worked, but I wanna have a class component, since makeStyles is hook api and I can't use that inside classes. so  useed styled components API. (https://material-ui.com/styles/basics/) but still didn't work for me what should I do ?
